test/apps/iodemo/run_io_demo.sh

```diff
@@ -222,9 +222,12 @@ parse_args()
 
 check_params()
 {
-	if [ ! -x "${iodemo_exe}" ]
+	if [ -z "${iodemo_exe}" ]
```

Comment:
```-z``` will check if var has value, while we want to check that variable is executable (iodemo_exe)